digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/asset/asset.scala

```diff
@@ -978,21 +978,22 @@ case object CyclingAndWalking extends AssetTypeInfo { val typeId = 440; def geom
 case object Lanes extends AssetTypeInfo { val typeId = 450; def geometryType = "linear"; val label = "Lanes" ; val layerName = "lanes"; val nameFI = "Kaista"}
 
 
-object AutoGeneratedValues {
-  val allAutoGeneratedValues =
-    Seq(
-      "dr1conversion",
-      "dr1_conversion",
-      "automatic_correction",
-      "excel_data_migration",
-      "automatic_generation",
-      "vvh_generated",
-      "vvh_modified",
-      "vvh_mtkclass_default",
-      "auto_generated_lane",
-      "start_date_importer"
-    )
+object AutoGeneratedValues extends Enumeration {
```

Comment:
Hyym kyl sen näin saa tehtytä vaikka olis sen voinut tehdä samaa tyyliin kuin noi muutkin enum vaikka FunctionalClass tuolla rivillä 79 tässä tiedostossa. Vähän silmiin pistää että tää tehty eri tavalla kuin noi muuut mut ei nyt maailmaa kaada.

Reply:
Tosta enum nimestä vois poistaa ton monikko s

Reply:
Saisko LinearAssetTypes.generatedInUpdate sisällytettyä tänne ja tää vois nimetä johonkin joka viitttaa että kyseessä on systeemin joku sisäinen prosessi.

Reply:
Jos löytyy muitakin tunnisteita niin vois keskittää tänne myös

Reply:
Sitten voitaisiin pitää toi allAutoGeneratedValues